creator/cli.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
```suggestion
```

This can be left out this time. Because of the package layout of `creator`, the only proper way to run the `cli` module is with the `-m` flag as in `python3 -m creator.cli`.

(If someone tries to run directly via `python3 creator/cli.py` or `./creator/cli.py`, the script won't find the modules in the `creator` package, which is expected.)